bump_rpm_packaging

```diff
@@ -12,23 +12,22 @@ git checkout -b rpm/release-$FULLVERSION $PACKAGING_GIT_REMOTE/$REMOTE_BRANCH
 for package in $RPM_PACKAGES ; do
 	(
 		cd packages/foreman/$package
+
+		spec_file="${package}.spec"
+		spectool --list-files $spec_file | cut -d' ' -f2 | grep http | xargs --no-run-if-empty -n 1 basename | xargs --no-run-if-empty git rm
+
 		if [[ $FULLVERSION == *-RC* ]] ; then
 			if [[ FULLVERSION == *-RC1 ]] ; then
 				echo "TODO: bump release"
```

Comment:
Side note: if we used `rc1` instead of `RC1` it would lexically sort and the release bump is not needed. However, `RC1 < nightly`.